docs/modules/nexus/pages/index.adoc

```diff
@@ -1,2 +1,40 @@
-= Nexus
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
+= Nexus 3
+
+Nexus 3 is an open source repository management platform developed by Sonatype to host and provide different registry and repository formats to store software artifacts.
+
+This repository contains the logic and configuration for a central Nexus 3 repository instance to store build artifacts, provided by Jenkins jobs or provide access to shared and proxied repositories.
+
+== Setup
+
+The OpenShift templates are located in `ocp-config` and can be compared with the OC cluster using https://github.com/opendevstack/tailor[tailor]. For example, run `cd ocp-config && tailor status` to see if there is any drift between current and desired state.
+
+You are also able to use the script `prepare-nexus.sh` located in the `infrastructure-setup/scripts` folder of this repository.
+You will have to customize the script to match your target environment.
+
+If you have a running Nexus 3 instance and you only want to add scripted resources, take a look at the `create-nexus-resources.sh` script. +
+Inside this script you will find curl calls, that are used to run scripts in the Nexus 3 scripting API. +
+The called scripts are located in the `json` subfolder and can be customized by your need. +
```

Comment:
There seem to be some stray `+` signs at the end of the lines?

Reply:
The `+` is intended. It marks a linebreak in ASCIIDOC